src/registrar/assets/js/get-gov.js

```diff
@@ -834,3 +834,436 @@ function hideDeletedForms() {
 (function cisaRepresentativesFormListener() {
```

Comment:
when making a new request I saw this pop up in the console in Chrome
![image](https://github.com/cisagov/manage.get.gov/assets/109625347/a617a313-a967-43eb-8811-f23aa06d5319)

Reply:
@rachidatecs going to tag you on this just so it isn't lost in all the other messages, feel free to just react when read. If this can't be solved in this ticket we probably should document this as a follow on ticket.

Reply:
Whoa! The IIFE is running on all page. Nice catch.

Reply:
Fixed